hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/net/TestDFSNetworkTopology.java

```diff
@@ -64,37 +64,42 @@ public void setupDatanodes() {
     final String[] racks = {
         "/l1/d1/r1", "/l1/d1/r1", "/l1/d1/r2", "/l1/d1/r2", "/l1/d1/r2",
 
-        "/l1/d2/r3", "/l1/d2/r3", "/l1/d2/r3",
+        "/l1/d2/r3", "/l1/d2/r3", "/l1/d2/r3", "/l1/d2/r3",
 
         "/l2/d3/r1", "/l2/d3/r2", "/l2/d3/r3", "/l2/d3/r4", "/l2/d3/r5",
 
         "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1",
-        "/l2/d4/r1", "/l2/d4/r1"};
+        "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r2",
+        "/l3/d5/r1", "/l3/d5/r1", "/l3/d5/r2"};
```

Comment:
nit: Add a blank line before every new rack, aka
```
  "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1",
  "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r2",
  "/l3/d5/r1", "/l3/d5/r1", "/l3/d5/r2"};
```
can be 
```
  "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r1",
  "/l2/d4/r1", "/l2/d4/r1", "/l2/d4/r2",
  
  "/l3/d5/r1", "/l3/d5/r1", "/l3/d5/r2"};
```

Same to the `hosts`, please  make the last three hosts in a new line so that `racks`, `hosts`, and `types` can be easily read with eyeballs.